build/configs/artik053/elf/defconfig

```diff
@@ -22,6 +22,8 @@ CONFIG_FRAMEWORK_DIR="../framework"
 CONFIG_TOOLS_DIR="../tools"
 # CONFIG_BUILD_FLAT is not set
 CONFIG_BUILD_PROTECTED=y
+CONFIG_APP_BINARY_SEPARATION=y
+CONFIG_NUM_APPS=2
```

Comment:
do we need hard code this value for fault manager?

Reply:
I have enabled the Application binary separation, and whichever macros got reflected due to that I have added in this defconfig